src/useIsStrictMode.ts

```diff
@@ -4,16 +4,21 @@ import * as React from "react";
  * @returns Current react fiber being rendered
  */
 export const getCurrentOwner = () => {
+  // If you use “React” imported directly, the following code supporting different
+  // React versions will cause errors when building with the module bundler,
+  // so temporarily assign it to a variable before use.
+  const react = React;
+
   try {
     // React 19
     // @ts-ignore - using react internals
-    return React.__CLIENT_INTERNALS_DO_NOT_USE_OR_WARN_USERS_THEY_CANNOT_UPGRADE.A.getOwner();
+    return react.__CLIENT_INTERNALS_DO_NOT_USE_OR_WARN_USERS_THEY_CANNOT_UPGRADE.A.getOwner();
```

Comment:
@tumugin good catch 👍 However, I think that we should use an existing trick (that is widely used for `useInsertionEffect`) with string concatenation:

<img width="888" alt="image" src="https://github.com/user-attachments/assets/a04d641c-2f21-47dc-aad1-989664c6a747">

Reply:
@layershifter

Thank you very much! I fixed to use this existing trick.

However, I was simply trying to use the `React['__' + "CLIENT_INTERNALS_DO_NOT_USE_OR_WARN_USERS_THEY_CANNOT_UPGRADE'].A.getOwner()` way, but `tsup` seems to optimize this code to `React['__CLIENT_INTERNALS_DO_NOT_USE_OR_WARN_USERS_THEY_CANNOT_UPGRADE'].A.getOwner()`.

So I used `React['__'.concat('CLIENT_INTERNALS_DO_NOT_USE_OR_WARN_USERS_THEY_CANNOT_UPGRADE')].A.getOwner()` to avoid optimizing.